src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java

```diff
@@ -529,4 +531,22 @@ public void siControllaLaTabellaDelegheACaricoDellImpresa() {
     public void siControllaLaTabellaDelegatiDallImpresa() {
         delegatiImpresaSection.checkTabellaDelegheDellImpresa();
     }
+
+    @And("Si inserisce un codice della delega a carico dell impresa errato nella modale")
+    public void siInserisceUnCodiceDellaDelegaACaricoDellImpresaErratoNellaModale() {
+        logger.info("Si inserisce un codice della delega errato nella modale");
+        deleghePagoPATest.siSceglieOpzioneAccetta();
+        String verificationCode = "00000";
+        deleghePGPagoPAPage.inserimentoCodiceDelegaACaricoDellImpresaAPI(verificationCode);
+        nellaSezioneDelegheSiCliccaSulBottoneConfermaCodice();
```

Comment:
Try to avoid using feature steps whenever possible, as they are intended for cucumber files only. Instead, directly use the real method: 
https://github.com/pagopa/pn-frontend-e2e/blob/ffdbfbd7bf209f0ae62e5056c13c9a861891495e/src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java#L169-L171